app/konnect/dev-portal/audit-logging/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Audit Logging in Dev Portal
+content_type: concept
+---
+
+Dev Portal audit logs are set up and managed separately from org-wide {{site.konnect_short_name}} audit logs. For more information about how to configure audit logging for {{site.konnect_short_name}}, see [Set up an audit log webhook for Dev Portal](/konnect/dev-portal/audit-logging/webhook/).
```

Comment:
The "how to configure" URL link for Audit Log for Konnect platform refers to Dev Portal audit log. Can we replace it with the URL for Konnect platform please?

Reply:
Good catch! I'll fix that